index.html

```diff
@@ -19,6 +19,16 @@
       </nav>
     </header>
 
+    <header class="header-desktop">
+      <nav>
+        <ul>
+          <li><a href="#project-list">Portfolio</a></li>
+          <li><a href="#about-me">About</a></li>
+          <li><a href="#get-in-contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
```

Comment:
I think you should only have one header and change the display of the internal elements. I could be wrong you can check up on this